### PR TITLE
ci: add py-13 tests and GHA dependabot

### PR DESCRIPTION
**Pull request by iloveitaly**

### CODE REVIEW

**Comment by alex-oleshkevich** [comment]
I would prefer not changing the formatting.

File: .tool-versions
Comment (alex-oleshkevich) [inline .tool-versions:1]: Let's not commit this file.

Comment (iloveitaly) [reply .tool-versions:1]: removed

**Comment by iloveitaly** [comment]
> I would prefer not changing the formatting.

yeah, this is automatically done by VS code on my end :(

**Comment by iloveitaly** [comment]
eliminated bad formatting!

**Comment by alex-oleshkevich** [comment]
It was not bad, just unexpected to see it in this PR. What is that dependabot config for?

**Comment by alex-oleshkevich** [comment]
Let's focus on 3.13 only w/o extra features/configurations.

**Comment by iloveitaly** [comment]
> It was not bad, just unexpected to see it in this PR. What is that dependabot config for?

to keep GHA versions up to date: I find that GHA will fail out over time if action verions are not updated.

**Pull request merged**